README.md

```diff
@@ -15,12 +15,24 @@ Virtual Schemas can be also described in known terms as `External Tables` or `Fo
 
 ## Version Requirements
 
-Virtual Schema  | Required Java Version | Lifecycle
-----------------|-----------------------|--------------------------------
-4.x.x           |                    11 | supported, active development
-3.x.x           |                    11 | supported
-2.x.x           |                     9 | discontinued
-1.x.x           |                     8 | discontinued
+The version of a Virtual Schema consists of two parts. The first part is the version of the [Virtual Schema JDBC](https://github.com/exasol/virtual-schema-common-jdbc/releases) in which the actual Virtual Schema is based on, and the second part is the version of the Virtual Schema itself.
+
+Thus the jar filename of a Virtual Schema release complies to the following format: `virtual-schema-dist-<Virtual Schema JDBC Version>-<Virtual  Schema Name>-<Virtual Schema Version>.jar`.
+
+For example, the jar filename of the [Oracle Virtual Schema 2.0.0 release](https://github.com/exasol/oracle-virtual-schema/releases/tag/2.0.0), which is based on the [9.0.1 version of Virtual Schema JDBC](https://github.com/exasol/virtual-schema-common-jdbc/releases/tag/9.0.1), is `virtual-schema-dist-9.0.1-oracle-2.0.0.jar`.
+
+The the version of the Virtual Schema JDBC in which a Virtual Schema is based on also tells us whether it is in supported or discontinued, as shown in the following table:
+
+Virtual Schema JDBC Version | Required Java Version | Lifecycle
+----------------------------|-----------------------|--------------------------------
+9.x.x                       |                    11 | supported, active development
+8.x.x                       |                    11 | supported
+7.x.x                       |                    11 | discontinued
+6.x.x                       |                    11 | discontinued
+5.x.x                       |                    11 | discontinued
+3.x.x                       |                    11 | discontinued
+2.x.x                       |                     9 | discontinued
+1.x.x                       |                     8 | discontinued
```

Comment:
Please update your Virtual Schema to a supported version before writing tickets or contacting the Exasol Support.